atomics/T1098.004/T1098.004.yaml

```diff
@@ -0,0 +1,19 @@
+---
+attack_technique: T1098.004
+display_name: SSH Authorized Keys
+
+atomic_tests:
+- name: Modify SSH Authorized Keys
+  description: |
+    Modify contents of <user-home>/.ssh/authorized_keys to maintain persistence on victim host. 
+    If the user is able to save the same contents in the authorized_keys file, it shows user can modify the file.
+  supported_platforms:
+    - macos
+    - linux
+  executor:
+    name: sh
```

Comment:
Id be explicit here and call out `bash` as not all shells support the `$()` operator.

Reply:
Thanks @rc-grey. Changed to bash executor.